.golangci.yml

```diff
@@ -1,31 +1,43 @@
 linters:
   disable-all: true
   enable:
+  - asasalint
   - asciicheck
   - bodyclose
+  - containedctx
+  - decorder
   - depguard
   - dogsled
   - errcheck
+  - errchkjson
+  - execinquery
   - exportloopref
   - gci
   - goconst
   - gocritic
+  - gocyclo
   - godot
   - gofmt
+  - goheader
   - goimports
   - goprintffuncname
   - gosec
   - gosimple
   - govet
+  - grouper
   - importas
   - ineffassign
+  - logrlint
+  - maintidx
```

Comment:
This one is like `gocyclo` and we already having that in the above list.

Reply:
Removed (avoid duplication), fixed.